microsoft/PowerToys
# Basic settings for Plugin Manager

## Summary of the Pull Request

**What is this about:**
It adds basic controls to settings in order to manage plugins. It should be easier to review it in small chunks. Take into account that accessibility is not fully implemented. For example, tab navigation does not work correctly for ListView.

**What is include in the PR:** 
- `PowerLauncherPluginViewModel`
- Move `isDark` mode functionality to App
-  Added two value converters: `BooleanInvertConverter`, `BooleanToVisibilityConverter`. `BooleanToVisibilityConverter` is present in WPF but not in UWP.
- Custome collapse functionality. It seems a lot easier than redefining `ListViewItem`
- Pass `isDark` action to `PowerLauncherPluginViewModel` as it is in library project. It will be simplified once we merge two projects

**How does someone test / validate:** 
- Check if `PowerToys Run` contains plugins settings
- Try to disable/enable, check/uncheck 'Include in global result', and editing 'Direct activation phrase' and verify if it propagates to PT Run
- Take into account know issues https://github.com/microsoft/PowerToys/issues/9653

## Quality Checklist

- [X] **Linked issue:** #5273

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
